Packs/Base/Scripts/CreateIndicatorRelationship/CreateIndicatorRelationship.py

```diff
@@ -0,0 +1,198 @@
+from typing import Tuple
+
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+
+import traceback
+
+BRAND = "XSOAR"
+PAGE_SIZE = 2000
+
+
+# --------------------------------------------------- Helper functions---------------------------------------------
+
+
+def find_indicators_by_query(query: str) -> List[dict]:
+    indicators: List[dict] = []
+    search_indicators = IndicatorsSearcher()
+
+    # last_found_len should be PAGE_SIZE (or PAGE_SIZE - 1, as observed for some users) for full pages
+    fetched_indicators = search_indicators.search_indicators_by_version(query=query, size=PAGE_SIZE).get('iocs')
+    while fetched_indicators:
+        # In case the result from searchIndicators includes the key `iocs` but it's value is None
+        fetched_indicators = fetched_indicators or []
+
+        # save only the value and type of each indicator
+        indicators.extend({'entity_b': ioc.get('value'), 'entity_b_type': ioc.get('indicator_type')}
+                          for ioc in fetched_indicators)
+        fetched_indicators = search_indicators.search_indicators_by_version(query=query, size=PAGE_SIZE).get('iocs')
+
+    return indicators
+
+
+def remove_existing_entity_b_indicators(args: dict) -> list:
```

Comment:
it is unclear what this function dose and why it is being called.
consider changing name or add a description to it in the doc string.

Reply:
added docstrings